release-notes/2.0/2.0.0.md

```diff
@@ -39,6 +39,16 @@ New features over the 1.1 release of Microsoft.Data.SqlClient:
 - [Username support for Active Directory Interactive mode](#username-support-for-active-directory-interactive-mode)
 - [Order Hints for SqlBulkCopy](#order-hints-for-sqlbulkcopy)
 
+## Breaking Changes in v2.0
+
+Breaking changes over the 1.1 release of Microsoft.Data.SqlClient:
+- The driver will now perform Server Certificate validation when TLS encryption is enforced by the target Server, which is the default for Azure connections [#391](https://github.com/dotnet/SqlClient/pull/391)
+- `SqlDataReader.GetSchemaTable()` now returns empty `DataTable` instead of returning `null` [#419](https://github.com/dotnet/SqlClient/pull/419)
+- Updated driver to perform decimal scale rounding to match SQL Server behavior [#470](https://github.com/dotnet/SqlClient/pull/470)
+- For .NET Framework applications consuming **Microsoft.Data.SqlClient**, the `SNI.dll` files previously downloaded to the `bin\x64` and `bin\x86` folders are now named `Microsoft.Data.SqlClient.SNI.x64.dll` and `Microsoft.Data.SqlClient.SNI.x86.dll` and will be downloaded to the `bin` directory, to support auto-loading in the application process [#570](https://github.com/dotnet/SqlClient/pull/570). _This change is not going to impact client applications unless a direct reference has been made to `SNI.dll` or the x86 and x64 folders._
+- Modified enclave provider interface `SqlColumnEncryptionEnclaveProvider` to be internal [#602](https://github.com/dotnet/SqlClient/pull/602) - _This change is not likely to impact customer applications since secure enclaves is a relatively new feature and they would have had to implement their own enclave provider, which is not a trivial task_.
```

Comment:
Not only SqlColumnEncryptionEnclaveProvider but also some other related classes to be internal